README.md

```diff
@@ -57,6 +58,17 @@ because for Mac, it requires performance boost with Docker Sync. Refer to
 `docker-compose.linux.yml` or `docker-compose.mac.yml` to `docker-compose.override.yml`
 in order to get your `ruby` workspace volumes mapped to your host machine.
 
+<a name="prerequisites"></a>
```

Comment:
I think if you removed this `Pre-requisites` section, I can merge the rest in. What do you think?

Reply:
Actually the main problem was the .env file, the reason i created this. But looks like you already fixed it.

So I guess we don't have to create the folder manually ?